### ISSUE TITLE
Design the website

### ISSUE THREAD

**Issue by swapagarwal** [swapagarwal/awesome-eli5#19]
We shall be hosting this at `eli5.tech`

What extra features would you like to see on the website?

One example I can think of is the ability to filter the explanations by some sort of subject-level segregation. Feel free to share more!

Also, it'd be great if someone volunteers to get the initial design of the website in place. Cheers!

**Comment by DerekCresswell** [comment]
I'm a web dev irl and could help out!
What were you thinking to build it up with?

**Comment by swapagarwal** [comment]
I'm fine with anything. What stack do you prefer (or maybe something new that you want to try out)?

**Comment by swapagarwal** [comment]
@DerekCresswell Do you want to take an initial shot at the rough design?

**Comment by amansrivastava905** [comment]
I can create a simple responsive website using react, styled-components & a bit of material-UI.
It may have features like sorting based upon topic,
to contribute a new item, you simply have to update the data.js file which will store content in form of objects.
If you have thought of any other flow, please comment. Will try to incorporate them.

I will share updates here.
